### PR TITLE
SignUp 

### PR DESCRIPTION
**Pull request by venkykatta**
**Signup page to different routes.**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,91 @@
+alembic==1.5.3
```
Comment (su66uu) [inline requirements.txt]: It seems you are not using all these packages. Can you remove unwanted packages?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    return render_template('signup.html')
+
+@app.route('/signup', methods=['POST'])
+def manager_page():
+    
+    role = request.form['role']
+    if role == "manager":
+        return render_template('manager.html')
+    else:
+        return render_template('chef.html')
```
Comment (su66uu) [inline main.py]: This is beautiful ❤️

**Review by su66uu — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (venkykatta) [reply requirements.txt]: Not sure! What I'm using in the packages in the local repository. 
Definitely, I remove the unnecessary packages in the local repository

File: main.py (hunk 2 above)
Comment (venkykatta) [reply main.py]: Thank You!

File: requirements.txt (hunk 1 above)
Comment (su66uu) [reply requirements.txt]: You did not understand how using packages works. Read up on it. Try to create the virtual environment first before start working on anything.

Comment (venkykatta) [reply requirements.txt]: Definitely, I will create a virtual environment.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.vscode
+env
```
Comment (su66uu) [inline .gitignore:2]: Why are we ignoring `env`?

File: main.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template, request, redirect
+from flask_mysqldb import MySQL
+ 
+app = Flask(__name__)
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_USER'] = 'root'
+app.config['MYSQL_PASSWORD'] = 'Shikhar@970'
+app.config['MYSQL_DB'] = 'mysqldb'
```
Comment (su66uu) [inline main.py:9]: These are Database connection secrets. Ideally, we should not expose them on codebase. Can you create a card to remove these secrets?

File: main.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+mysql = MySQL(app)
+
+@app.route("/")
+def signup_page():
+    return render_template("signup.html")
+
+
+@app.route("/signup", methods=["POST"])
+def manager_page():
```
Comment (su66uu) [inline main.py]: Why do you think `manager_page` name will suit for this function name?

File: main.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+    role = request.form["role"]
+    if role == "manager":
+            if request.method == "POST":
+                user_data = request.form
+                Name = user_data['name']
+                email = user_data['mail']
+                Password = user_data['password']
+                Role = user_data['role']
```
Comment (su66uu) [inline main.py]: Some variable names are starting with capital letters. Can you make all of them to small letters?

Comment (su66uu) [reply main.py]: Read this when you get free time https://github.com/google/styleguide/blob/gh-pages/pyguide.md

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+PyYAML==3.12
+requests==2.18.4
+requests-unixsocket==0.1.5
+simplejson==3.13.2
+ufw==0.36
+urllib3==1.22
+usb-creator==0.3.3
+virtualenv==15.1.0
+Werkzeug==1.0.1
+xkit==0.0.0
```
Comment (su66uu) [inline requirements.txt]: Still I see so many unused packages.

**Review by su66uu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (venkykatta) [reply .gitignore:2]: It shows a lot of uncommitted files. That's why I place the **env**  file in **.gitignore**.

File: main.py (hunk 4 above)
Comment (venkykatta) [reply main.py:9]: Yeah! sure Subbu. I will create a Trello card for that name as _Database Secrets_, Is it good Subbu?

File: main.py (hunk 5 above)
Comment (venkykatta) [reply main.py]: At the beginning of this application, I focus on, finish the task within time, that's why I couldn't focus on the clean code. Definitely, I will write the meaningful function name here.

File: main.py (hunk 6 above)
Comment (venkykatta) [reply main.py]: Okay Subbu

File: requirements.txt (hunk 7 above)
Comment (venkykatta) [reply requirements.txt]: I fix this issue

File: main.py (hunk 4 above)
Comment (su66uu) [reply main.py:9]: `Load database secrets from environment variables` would be good

File: .gitignore (hunk 3 above)
Comment (su66uu) [reply .gitignore:2]: Is it `venv` or just `env`

**Review by su66uu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (venkykatta) [reply .gitignore:2]: That is ` env`

File: main.py (hunk 4 above)
Comment (venkykatta) [reply main.py:9]: That's the good name  for the story

File: .gitignore (hunk 3 above)
Comment (su66uu) [reply .gitignore:2]: I think using venv would be good. Because most of the developers use this convention

**Review by su66uu — CHANGES_REQUESTED** [review]

**Review by su66uu — APPROVED** [review]

**Pull request merged**